### ISSUE TITLE
design home page

### ISSUE THREAD

**Issue by 98sean98** [rcltech/owl#3]
Pre-development:
- run react app as it is locally to check for problems

Todo:
- add logos and images as part of assets
- explore and add necessary css libraries such as `material-ui` and `bootstrap`

home page should contain:
- a list of rooms for users to choose for booking
- each room has a collapsible timetable that shows availability for 24 hrs for the next 3 days

potential issues:
- mobile-oriented design might run into problems on desktop

possible solutions:
- use more flexible css properties such as `vh`

**Comment by 98sean98** [comment]
Error while running `npm run start`

```
./src/components/InfoSection.js
Module not found: Can't resolve '@material-ui/core' in '/home/seanchok/Documents/Programming/rctech/hall-booking-system/react-app/src/components'
```

Please make sure that things work first before pushing. I think Travis CI isn't too difficult to integrate, so far I'm still trying to make a simple test to make sure that the express-app runs correctly; I will add react-app build tests for branch commits too.

**Update**
This problem has been fixed.

**Comment by 98sean98** [comment]
Still need a lot of work before this branch can be merged. We need to create the timetable component as well first.

**Comment by 98sean98** [comment]
Please fix the `package.json` and include necessary packages. The current error I have now, even after doing `npm install` is:
```
Failed to compile.

./src/components/Timetable.js
Module not found: Can't resolve 'react-calendar' in '/home/seanchok/Documents/Programming/rctech/hall-booking-system/react-app/src/components'
```

Also, look into issue #12 to write configs in Travis CI so that every time we push react code, it is checked if it could be built successfully.

**Comment by 98sean98** [comment]
I noticed that the `testing` branch has been merged to this `home-page` branch. That should not have happened. I'm going to remove those test features in this branch, before it can be merged to `master`. Those features should not be in `master`.

**Problem has been fixed** refer to commit 91f1213
